lib/chromedriver/helper.rb

```diff
@@ -35,6 +35,20 @@ def update
       download true
     end
 
+    def compatibility
+      if File.exists?("/usr/lib64/libstdc++.so.6")
```

Comment:
If this file doesn't exist, does the problem not exist? Can the problem exist without this file being present?